yarn.lock

```diff
@@ -53,6 +53,19 @@
     jsesc "^2.5.1"
     source-map "^0.5.0"
 
+"@agoric/base-zone@^0.1.1-u20.0":
```

Comment:
this file got much larger but shouldn't. please adjust the versions. you could use `*` for '@endo/' versions so that the agoric deps imply them. the agoric deps should also be linked up to a tag like u20.